packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/modeledDataStreamTable/modeledDataStreamTable.tsx

```diff
@@ -111,7 +111,7 @@ export function ModeledDataStreamTable({
             </Button>
             <Button
               variant='primary'
-              disabled={collectionProps.selectedItems?.length === 0 || selectedWidgets.length === 0}
+              disabled={collectionProps.selectedItems?.length === 0 || selectedWidgets.length !== 1}
```

Comment:
note: this will eventually break when we allow for more than 1 selected widget
approving this since it will not break things at the moment